### PR TITLE
Refactor Samplers - Remove n for Sampler, WordSampler, Decile Sampler…

### PR DESCRIPTION
**Pull request by phenomenal-manish**
…, and Deceptive Sampler

*Issue #, if available:*

*Description of changes:*

Refactor Samplers - Remove n for Sampler, WordSampler, Decile Sampler, and Deceptive Sampler.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ndronen — DISMISSED** [review]
LGTM

File: setlexsem/generate/generate_sets.py [hunk 1]
```diff
@@ -187,6 +187,7 @@ def make_sets(
     config: Dict[str, Any] = {},
     number_of_data_points: int = 100,
     seed_value: int = 292,
+    random_state=None,
```
Comment (ndronen) [inline setlexsem/generate/generate_sets.py]: Shouldn't this be removed?

**Review by ndronen — COMMENTED** [review]
Needs some cleanup.

File: setlexsem/generate/generate_sets.py [hunk 2]
```diff
@@ -210,7 +211,7 @@ def make_sets(
     )
     all_sets = []
     for hp in make_hps_generator:
-        random_state = random.Random(seed_value)
+        # random_state = random.Random(seed_value)
```
Comment (BardiyaAk) [inline setlexsem/generate/generate_sets.py]: i'm worried about this, if we don't refresh the seed, the order of how hyperparameters are loaded can change the dataset we use.

It's okay for those cases that we read the dataset, but for those cases we/user wants to generate data on the fly, this affects things.

File: setlexsem/generate/generate_sets.py [hunk 3]
```diff
@@ -231,48 +232,48 @@ def make_sets(
...
-                    number_of_data_points,
-                    overwrite=overwrite,
-                )
-
-        except Exception as e:
-            logger.warning(f"Skipping: {e} / {sampler}")
-            continue
-
-    logger.info("Dataset is created!")
+# if __name__ == "__main__":
```
Comment (BardiyaAk) [inline setlexsem/generate/generate_sets.py]: why are we commenting this?

File: setlexsem/generate/prompt.py [hunk 4]
```diff
@@ -408,10 +409,10 @@ def get_prompt(A, B, prompt_config, add_roles=False):
     else:
         prompt = ""
 
-    # define the inputs and instruction
-    prompt += (
-        f"You are given two sets. Set A is ({A_str}). Set B is ({B_str})."
-    )
+        # define the inputs and instruction
```
Comment (BardiyaAk) [inline setlexsem/generate/prompt.py]: ?

Comment (BardiyaAk) [reply setlexsem/generate/prompt.py]: why is that?

File: setlexsem/generate/generate_sets.py (hunk 2 above)
Comment (phenomenal-manish) [reply setlexsem/generate/generate_sets.py]: I added it back in a new commit!

File: setlexsem/generate/generate_sets.py (hunk 3 above)
Comment (ndronen) [reply setlexsem/generate/generate_sets.py]: This is another change made during debugging. @phenomenal-manish, please restore the block of code.

File: setlexsem/generate/prompt.py (hunk 4 above)
Comment (phenomenal-manish) [reply setlexsem/generate/prompt.py]: corrected in second commit

File: setlexsem/generate/generate_sets.py (hunk 3 above)
Comment (phenomenal-manish) [reply setlexsem/generate/generate_sets.py]: Done

File: setlexsem/generate/generate_sets.py (hunk 1 above)
Comment (phenomenal-manish) [reply setlexsem/generate/generate_sets.py]: Removed

**Review by BardiyaAk — APPROVED** [review]
Looks good to me now, thanks a lot

**Pull request merged**